frontend/src/i18n/locales/en.yaml

```diff
@@ -199,10 +199,7 @@ upload:
     field-required: This field is required (cannot be empty)
     opencast-server-error: Opencast server error (unexpected response).
     opencast-unreachable: 'Network error: Opencast cannot be reached.'
-    jwt-expired: >
-      Internal cross-authentication error: due to your internet connection, the video was uploaded too slowly.
-      Currently, this application cannot deal with those cases. We are aware of this problem and
-      working on it! We are sorry for causing any inconvenience.
+    jwt-expired: 'Internal cross-authentication error: the video was uploaded too slowly.'
```

Comment:
I would remove the "too slowly" completely. The upload is not a problem anymore right? I think it would send a wrong signal to the user. So this error is shown exactly when any request done by the uploader results in a redirect (which in Opencast means not authenticated, sigh...). So I think I would rename this translation key to `jwt-invalid` and change the translation to just `Internal cross-authentication error`.

Reply:
Seems to fit well into this one, no? I mean the cases in which this error is thrown can stay, no? If redirect -> unauthorized -> JWT invalid somehow.